.travis.yml

```diff
@@ -1,15 +1,12 @@
 sudo: required
 dist: trusty
```

Comment:
you can probably remove this while you're here (it slows down builds and shouldn't be necessary anymore)